tests/test_fsentry.py

```diff
@@ -330,6 +330,15 @@ def test_serialize_structmap_child_empty(self):
         assert len(el.attrib) == 2
         assert len(el) == 0
 
+    def test_serialize_structmap_directory_admid(self):
+        """It should add the ADMID attribute to directories with amdsecs."""
+        d = metsrw.FSEntry("objects", type="Directory")
+        f = metsrw.FSEntry("file.txt", file_uuid=str(uuid.uuid4()))
```

Comment:
I can see across the module/tests we've adopted a lot of short-hand for variables! One for the future to fix-up maybe. I think it's good you're following the style of the code previously adopted though.